### PR TITLE
Docs: Add code-block wrappers to code examples

### LINKED ISSUE #79028: Docs: Add code-block wrappers to code examples in guide_alicloud.rst
### Summary

**Problem**:
Throughout the Ansible docs, there are instances where example code is preceded with a lead-in sentence ending in `::`.
Translation programs then attempt to translate this code, which we don't want.

**Solution:**
Enclose code  in a `.. code-block:: <lexer>` element, so that translation processes know to skip this content.
For a list of allowed values for _`<lexer>`_ , refer to [Syntax highlighting - Pygments](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#syntax-highlighting-pygments).

**Scope:**
In the  `guide_alicloud.rst` file in the Scenario Guides (`docs/docsite/rst/scenario_guides`), there are 2 instances where a lead-in sentence ends with `::`. Use the following `grep` command to identify the line numbers:

```
$ grep -rn --include "*.rst" "^[[:blank:]]*[^[:blank:]\.\.].*::$" .
```

**Example:**

Before:
```
Before running ``ansible-playbook``, run the following command to enable logging::

    export ANSIBLE_LOG_PATH=~/ansible.log

```

After:
```
Before running ``ansible-playbook``, run the following command to enable logging:

.. code-block:: shell

    export ANSIBLE_LOG_PATH=~/ansible.log

```

This problem has been addressed in some other guides; view these merged PRs to help get you started:
- Network Guide: [#75850](https://github.com/ansible/ansible/pull/75850/files) 
- Developer Guide: [#75849](https://github.com/ansible/ansible/pull/75849/files) 

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/scenario_guides/guide_alicloud.rst

### Ansible Version

```console
n/a
```

### Configuration

```console
n/a
```

### OS / Environment

n/a

### Additional Information

When example code is enclosed within a code-block element, translation programs do not attempt to translate the code.

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by muditchoudhary**
##### SUMMARY

Add code-block wrappers to code examples.
Fixes: #79028

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/scenario_guides/guide_alicloud.rst

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/guide_alicloud.rst [hunk 1]
```diff
@@ -30,7 +32,9 @@ Authentication
 You can specify your Alicloud authentication credentials (access key and secret key) by passing them as
 environment variables or by storing them in a vars file.
 
-To pass authentication credentials as environment variables::
+To pass authentication credentials as environment variables:
+
+.. code-block:: yaml
```
Comment (ariordan-redhat) [inline docs/docsite/rst/scenario_guides/guide_alicloud.rst]:
```suggestion
.. code-block:: shell
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
@muditchoudhary 
Could you please modify one of the lexers to `shell`?

Otherwise, the changes look good to me.

**Comment by muditchoudhary** [comment]
Done @ariordan-redhat see if it's good

**Review by ariordan-redhat — APPROVED** [review]
Built doc looks good to me.

**Comment by ariordan-redhat** [comment]
Thank you for contributing to the Ansible docs, @muditchoudhary !

**Comment by muditchoudhary** [comment]
> Thank you for contributing to the Ansible docs, @muditchoudhary !

Welcome :)

Thanks too for being helpful

**Pull request merged**